pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	"github.com/eriknordmark/netlink"
 	"github.com/google/go-cmp/cmp"
+	zconfig "github.com/lf-edge/eve/api/go/config"
```

Comment:
Why rename this to `zconfig` and not just leave it as the default `config`? It doesn't appear to conflict with another import.

Reply:
Was used similarly in other places.. hence did the same. IIUC - config is too generic a word.. zconfig clearly indicatesit is from the API..

Reply:
OK. My preference is to stick to config, but I won't push too hard on it.

Reply:
Issue is that we have local variables in many files called config (and status). That seems to conflict with the package name.

Reply:
Got it. I only looked at package names. Go is quite good about figuring out vars vs packages, but sometimes it cannot tell. Good to leave it then.